hasher-matcher-actioner/webapp/src/utils/ImageUtils.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ */
+
+import React, {useState} from 'react';
+import {Image} from 'react-bootstrap';
+
+export const BlurUntilHoverImage =
```

Comment:
blocking: It looks like from the video in the test plan that when loading the image theres a split second where it's not blurred (Am I seeing things?). This seems to somewhat defeat the purpose of blurring. Any idea why this is happening?

Reply:
It has to do with the page layout, when you click 'details' your mouse is over the image when it loads... looked at a bunch of different solutions to avoid this but most are unintuitive unfortunately.

Reply:
Leaving this for now, fixes like 'delaying mouse events' or having another state variable made for a less than ideal UX imo, possible solutions to consider in future
- reorganization of the page to prevent the button from the last page from falling right on the image
- Adding button to blur and unblur (will likely end up adding anyway)